nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -240,9 +240,7 @@ def _test_convolution(tensor_in_sizes, filter_in_sizes,
 def test_forward_convolution():
     if is_gpu_available():
         _test_convolution([4, 176, 8, 8], [1, 1, 176, 32], [1, 1], [1, 1], 'SAME', 'NCHW')
-        _test_convolution([4, 19, 17, 17], [3, 3, 19, 19], [1, 1], [2, 2], 'VALID', 'NCHW')
         _test_convolution([4, 124, 17, 17], [1, 1, 124, 19], [1, 1], [1, 1], 'SAME', 'NCHW')
-        _test_convolution([4, 12, 17, 17], [3, 3, 12, 32], [1, 1], [2, 2], 'VALID', 'NCHW')
```

Comment:
Any reason why padding=='VALID' tests are removed?

Reply:
:+1:  enabled the test cases with fixes. 

After correcting GPU flag check function some cases become invalid.

Reply:
The GPU check flag fix in this PR caused some test cases invalid for tensorflow GPU.
Fixed all of them now :+1: